### DISCUSSION TITLE
Could this be pushed to realtime?

### DISCUSSION THREAD

**Discussion by maximxlss** [suno-ai/bark#274] [General]
This thing is so awesome, but my application is realtime and so not applicable. I was thinking if at some point it could be realtime on high-end hardware (3090 ti in my case) or if it's possible to code and train similar but smaller model on data with one language to get good results. What do you think?

**Comment by gkucsko** [comment]
out of curiosity what would that mean for you. would the text be available in advance and you only care about generating it 'faster than realtime' or would the words be streaming in one-by-one?

**Reply by maximxlss** [reply]
My application doesn't have the text in advance and requires minimal latency, so I need either streaming audio or really, really fast generation. My problem currently is that there is little pretrained TTS systems for Russian. The best one I guess is Silero, but I hate the fact that it doesn't natively support english inserts or any kind of symbols, and just ignores them.

**Reply by SlashWong** [reply]
跟我的需求一样，离线转换太慢了，多数的场景是实时或流式的text2speech场景。

**Comment by scofieldWyq** [comment]
could this support generating voice stream data ?

**Comment by wenchocho2** [comment]
emmmmm
